cdap-ui/app/cdap/components/PipelineList/DraftPipelineView/DraftTable/SortableHeader.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { connect } from 'react-redux';
+import { setSort } from 'components/PipelineList/DraftPipelineView/store/ActionCreator';
+import SortableHeaderView from 'components/PipelineList/SortableHeaderView';
+
```

Comment:
Looks like this same component we use for deployed table view as well right? Because there is no store associated with it can we reuse the one from deployed pipeline view?

Reply:
That's exactly what this is doing. This is just connecting the component with the appropriate actions and store

Reply:
the shared functional component is at the PipelineList level, and the component in both Deployed and Drafts location are just connecting the component to the stores